app/views/dashboard/assessmentDetails.scala.html

```diff
@@ -111,28 +111,28 @@
     </div>
 }
 
+@noRateableValueContent = {
+    <p>@messages("rv.na.p.1")</p>
```

Comment:
p tags need govuk-body class

Reply:
I think not in this case as this block is used inside a govUkDetails, but will check